modules/ztp-syncing-new-configmap-changes-to-existing-pgt-crs.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-syncing-new-configmap-changes-to-existing-pgt-crs_{context}"]
+= Syncing new ConfigMap changes to existing PolicyGenTemplate CRs
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have created a `PolicyGenTemplate` CR that pulls information from a ConfigMap CR using hub templating.
+
+.Procedure
+
+. Update the contents of your `ConfigMap` CR, and apply the changes in the hub cluster.
+
+. To sync the contents of the updated `ConfigMap` CR to the deployed policy, do either of the following:
+
+.. Optional: Delete the existing policy and recreate it using the `PolicyGenTemplate` CR.
+
+... Delete the policy, for example:
++
+[source,terminal]
+----
+$ oc delete policy <policy-name> -n <policy-namespace>
+----
++
+... Re-apply the `PolicyGenTemplate` CR:
++
+[source,terminal]
+----
+$ oc apply -f sriov-pgt-example.yaml
+----
+
+.. Optional: Apply a special annotation `policy.open-cluster-management.io/trigger-update` to the policy with a different value every time when updating the `ConfigMap`. For example, run the following command:
++
+[source,terminal]
+----
+$ oc label <policy-name> -n <policy-namespace> policy.open-cluster-management.io=trigger-update
+----
+
+... If it exists, delete the existing `ClusterGroupUpdate` CR that contains the policy. Create a new `ClusterGroupUpdate` CR to apply the updated `ConfigMap` changes. For example:
++
+[source,terminal]
+----
+$ oc delete clustergroupupgrade <cgu-name> -n <cgu-namespace>
+----
+
+... Re-apply the `PolicyGenTemplate` CR to regenerate the `ClusterGroupUpdate` CR. For example, run the following command:
++
+[source,terminal]
+----
+$ oc apply -f sriov-pgt-example.yaml
```

Comment:
This doesn't look right to me. After the label is updated in PGT, should the next step be "oc apply -f sriov-pgt-example.yaml" to update the policy? then create a new CGU to rollout the change made in configmap